sdk/core/Azure.Core/tests/CoreResponseClassifierTests.cs

```diff
@@ -14,7 +14,7 @@ public void ClassifiesSingleCodeAsNonError()
             // test classifiers for each of the status codes
             for (ushort nonError = 100; nonError <= 599; nonError++)
             {
-                CoreResponseClassifier classifier = new CoreResponseClassifier(new ushort[] { nonError });
+                StatusClassifier classifier = new StatusClassifier(new ushort[] { nonError });
```

Comment:
rename the test files to match the new names?

Reply:
Good catch - TY!